### PR TITLE
bookinfo doc fixes

### PR DESCRIPTION
**Pull request by krancour**
This PR complements changes in https://github.com/istio/istio/pull/7109

### CODE REVIEW

File: content/docs/reference/config/istio.networking.v1alpha3/index.html [hunk 1]
```diff
@@ -1392,8 +1392,8 @@ <h2 id="HTTPMatchRequest">HTTPMatchRequest</h2>
 <p>HttpMatchRequest specifies a set of criterion to be met in order for the
 rule to be applied to the HTTP request. For example, the following
 restricts the rule to match only requests where the URL path
-starts with /ratings/v2/ and the request contains a <code>cookie</code> with value
```
Comment (geeknoid) [inline content/docs/reference/config/istio.networking.v1alpha3/index.html]: All changes in this file need to be undone and instead applied to the appropriate proto file in the istio/api repo. Once the proto is updated, this HTML file will automatically get regenerated.

Thanks.

**Comment by krancour** [comment]
Updated as requested. Also-- should there be a `release-1.0` branch for this project that I can target with this PR? Changes that _this_ PR documents are targeted at `release-1.0` in `istio/istio` and `istio/api`.

**Comment by geeknoid** [comment]
There is no release-1.0 branch in this repo yet. The branch will be created the day 1.0 is launched.

**Comment by krancour** [comment]
@geeknoid is this repo continuously published on merge to master? What I'm worried about is that I would not want to see the changes in this PR published prior to the 1.0 release since the changes to the bookinfo app that this PR documents are in the `release-1.0` branch.

Can you offer some guidance please on how to proceed?

**Comment by frankbu** [comment]
@krancour You should merge to master. The content will not be published to the istio.io site when you merge to master, but rather to preliminary.istio.io. It will be switched over when the release goes out.

**Comment by krancour** [comment]
@frankbu thanks for the explanation.

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 2]
```diff
@@ -90,9 +90,8 @@ If you look in the sample services, you can see that the productpage application
 def getForwardHeaders(request):
     headers = {}
 
-    user_cookie = request.cookies.get("user")
-    if user_cookie:
-        headers['Cookie'] = 'user=' + user_cookie
+    if 'user' in session:
+        headers['end-user'] = session['user']
```
Comment (krancour) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: These changes accurately reflect the code that is in the productpage service in the `release-1.0` branch.

File: content/docs/tasks/telemetry/distributed-tracing/index.md [hunk 3]
```diff
@@ -116,20 +115,21 @@ The reviews application (Java) does something similar:
...
                             @HeaderParam("x-ot-span-context") String xotspan) {
-  String r1 = "";
-  String r2 = "";
+  int starsReviewer1 = -1;
+  int starsReviewer2 = -1;
 
-  if(ratings_enabled){
-    JsonObject ratings = getRatings(user, xreq, xtraceid, xspanid, xparentspanid, xsampled, xflags, xotspan);
+  if (ratings_enabled) {
+    JsonObject ratingsResponse = getRatings(Integer.toString(productId), user, xreq, xtraceid, xspanid, xparentspanid, xsampled, xflags, xotspan);
```
Comment (krancour) [inline content/docs/tasks/telemetry/distributed-tracing/index.md]: These changes accurately reflect the code that is in the reviews service in the `release-1.0` branch.

**Comment by krancour** [comment]
From my perspective, this PR is ready for review then.

File: content_zh/docs/tasks/traffic-management/request-routing/index.md [hunk 4]
```diff
@@ -158,7 +158,7 @@ keywords: [traffic-management,routing]
 
 ## 理解原理
 
-在此任务中，您首先使用 Istio 将100%的请求流量都路由到了 BookInfo 服务的v1版本。 然后再设置了一条路由规则，该路由规则基于请求的 header（例如一个用户 cookie）选择性地将特定的流量路由到了 reviews 服务的v2版本。
+在此任务中，您首先使用 Istio 将100%的请求流量都路由到了 BookInfo 服务的v1版本。 然后再设置了一条路由规则，该路由规则基于请求的 选择性地将特定的流量路由到了 reviews 服务的v2版本。
```
Comment (rootsongjc) [inline content_zh/docs/tasks/traffic-management/request-routing/index.md]: This paragraph needs to be updated, I will do it later.

**Comment by geeknoid** [comment]
PS: Checkout the README file at the root of the repo to learn about preliminary.istio.io and how we manage versions of the site content.

**Comment by krancour** [comment]
Rebased.

**Comment by rshriram** [comment]
fix lint please

**Comment by krancour** [comment]
@rshriram not sure how those fancy quotes crept into this PR, but I fixed them just now.

**Pull request merged**